app/assets/stylesheets/welcome.scss

```diff
@@ -2,3 +2,40 @@
 // They will automatically be included in application.css.
 // You can use Sass (SCSS) here: http://sass-lang.com/
 $backdrop : #C9D6AE;
+
+.welcome {
+  background-image: image-url('skyscrapers.jpg');
+  background-repeat: no-repeat;
+  -webkit-background-size: cover;
+  -moz-background-size: cover;
+  -o-background-size: cover;
+  background-size: cover;
+  min-height: 100%;
+  min-width: 1024px;
+  }
+
+.top-bar {
+  position-left: 0;
+  position-right: 0;
+  width: 100%;
+}
+
+.fullWidth {
+   width: 100%;
```

Comment:
Line should be indented 2 spaces, but was indented 3 spaces<br>Properties should be ordered margin-left, margin-right, max-width, width